style.css

```diff
@@ -369,176 +561,476 @@ header .nav-menu ul li a:hover {
   transform: rotate(-45deg);
 }
 
-@media screen and (max-width: 480px) {
+/* ------------- Contact ---------------- */
+.contact {
+  display: flex;
+  justify-content: space-between;
+  padding: 100px 5%;
+}
+
+.contact .form-text {
+  width: 40%;
+}
+
+.contact form .form-container {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding: 0;
+}
+
+.footer-section .social-media-container {
+  margin-top: 50px;
+  list-style: none;
+  font-size: 30px;
+  display: flex;
+  padding: 0;
+}
+
+.footer-section .social-media-container .item .icon {
+  color: #505f79;
+}
+
+.footer-section .social-media-container .item a:hover {
+  color: #f55800;
+}
+
+.footer-section .item:not(:last-child) {
+  margin-right: 20px;
+}
+
+/* ============================================
+==============================================
+==============================================
+============================================== */
+
+@media screen and (min-width: 1080px) {
   body {
```

Comment:
- [ ] You should use `min-width: 768px;` as the previous reviewer suggested and as per the project requirement.
![desktop-vers-768-instruct](https://user-images.githubusercontent.com/44057863/167274739-284d7322-4f94-43e7-aecf-c28f57a7755c.png)